en/companion_computer/holybro_pixhawk_rpi_cm4_baseboard.md

```diff
@@ -72,10 +77,82 @@ The image below shows the wiring in greater detail.
 
 ![Image showing writing from the PM03D power module to the baseboard](../../assets/companion_computer/holybro_pixhawk_rpi_cm4_baseboard/baseboard_wiring_guide.jpg)
 
+# RPi CM4 Flash Guide
+
+### Please **Note:**
+
+* If you are using PX4, you will need to use PX4 version 1.13.1 or newer for PX4 to recognize this baseboard.
+* The fan does not indicate if the RPi CM4 is powered/running or not.
+* The power module plugged into Power1/2 does not power the RPi part. You can use the additional USB-C Cable from the PM03D power module to the CM4 Slave USB-C port.
+* The Micro-HDMI port is an output port.
+* Some RPi CM4 might not have Wifi device and therefore won’t connect automatically, unless you plug it into a router or a compatible Wifi dongle into the CM4 Host ports.
+
+### Flash EMMC <a href="#flash-emmc-2" id="flash-emmc-2"></a>
```

Comment:
FYI, I will remove all the heading links - you don't reference these anywhere so I assume they are copy-paste errors.